packages/test-generator/integration-test-templates/src/FormTests/DSCompositeDog.tsx

```diff
@@ -157,12 +145,9 @@ export default function () {
       <Heading>DataStoreFormUpdateCompositeDog</Heading>
       <View id="DataStoreFormUpdateCompositeDog">
         <DataStoreFormUpdateCompositeDog
-          compositeDog={dataStoreFormUpdateCompositeDogRecord}
+          id={compositeDogPrimaryKey}
           onSuccess={async () => {
-            const record = (await DataStore.query(CompositeDog, {
-              name: 'Yundoo',
-              description: 'tiny but mighty',
```

Comment:
Instead of modifying the existing test, can you add a new one?
So, just render this component below with a different `id`, like `DataStoreFormUpdateCompositeDog-byID` and add the test in the spec making sure the existing data shows up.

Reply:
ok. I modified this one because I thought there were plenty of test cases for passing in a model